packages/ui/src/composables/useElementBackground.stories.ts

```diff
@@ -0,0 +1,198 @@
+import { defineComponent, computed, ref } from 'vue'
+import { useElementBackground } from './useElementBackground'
+import { useCurrentElement } from './useCurrentElement'
+import { VaButton } from '../components/va-button'
+import UseElementBackgroundDummy from './UseElementBackgroundDummy.vue'
+import { within } from '@storybook/testing-library'
+import { sleep } from '../utils/sleep'
+import { expect } from '@storybook/jest'
+
+export default {
+  title: 'composables/useElementBackground',
+}
+
+export const Default = () => ({
+  components: { VaButton, UseElementBackgroundDummy },
+  data () {
+    return {
+      background: '#000',
+      color: null,
+    }
+  },
+  template: `
+  <UseElementBackgroundDummy
```

Comment:
I suggest we use "Dummy" component to test composable. I'm not sure how that might work for other cases, but connecting composable to story looks like a horrid mess to me, with no clear ins and outs. With separate component - it seems manageable.